.github/workflows/publish_release_reusable.yml

```diff
@@ -34,18 +34,12 @@ jobs:
       contents: write
     strategy:
       matrix:
-        os: [ubuntu-latest, macos-latest, windows-latest]
+        os: [ubuntu-latest]
         toolchain:
           - stable
     steps:
       - name: Checkout cedar-java
         uses: actions/checkout@v3
-      - name: Checkout cedar
```

Comment:
We only release once published to crates.io, so we can get rid of this in all branches